src/protobuf/WireFormats.proto

```diff
@@ -61,6 +61,7 @@ message DeployData {
 	string configManifest = 9;
 	int32 routerConfigSerializerId = 10;
 	string routerConfigManifest = 11;
+	int32 stashCapacity = 12;
```

Comment:
Addition to wire format - if this value is missing then the stash defaults back to being unbounded.